config/locales/pages/en.yml

```diff
@@ -4,3 +4,6 @@ en:
     page_not_found:
       body: You might want to double-check your link and try again. (404)
       header: The page you were looking for doesn’t exist.
+    page_took_too_long:
+      body: You might want to wait a few minutes and try again. (503)
```

Comment:
Can we change to: `Please wait a 2 or 3 minutes and try again.`

Reply:
Should this be: `Please wait 2 or 3 minutes and try again.`? (no `a` between `wait` and `2`)

Should we retain the `(503)` at the end, similar to the page not found message, or drop it?

Reply:
Actually, based on @monfresh's comments I'm going to cancel this request.